imsv-docs-astro/src/content/docs/guides/authentication.mdoc

```diff
@@ -154,3 +154,18 @@ be exchanged when the new access token expires.
 
 <!-- https://miro.com/app/board/uXjVNxzJMjE=/?moveToWidget=3458764581446931419&cot=14 -->
 ![Refresh Token Sequence Diagram](@assets/diagrams/refresh-token-sequence.svg)
+
+
+## Logout
+
+Logging out of the application involves invalidating the current access token to ensure it can no longer be used for accessing protected resources within the Immersve API.
```

Comment:
```suggestion
Logging out of the application involves invalidating the current session to ensure it can no longer be used for accessing protected resources within the Immersve API. All access tokens connected with the session will become unusable.
```

Reply:
done ✅